src/platforms/apple/common/dsym.mdx

```diff
@@ -47,41 +47,23 @@ Free subscription plans are limited to one App Store Connect source per project,
 
 </Alert>
 
-Setting up App Store Connect currently requires two separate credentials.
-
-1. **App Store Connect API Key**:
+Setting up an App Store Connect custom repository requires only the App Store Connect API Key.
 
 Follow the documentation on [Creating API Keys for App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api) and provide the _Issuer_, _Key ID_, and _Private Key_. The API Key needs to have the _Developer_ role in order to fetch build details.
 
-2. **iTunes Credentials**:
-
-An iTunes user and password are required, in addition to the normal App Store Connect API Key, due to limitations of the App Store Connect API. The App Store Connect API Key alone will only allow detection of new builds; an iTunes user is required to fetch the associated dSYM files.
-
-[Create a new Apple ID](https://support.apple.com/en-us/HT204316) specifically for this integration and provide its credentials. The user needs to have the _Developer_ role in the appropriate App Store Organization.
-
-<Alert level="warning">
-
-We highly recommend that you create a dedicated iTunes user account for the purpose of this integration, and that you **not** use a regular user account.
-
-Using the configured iTunes account on a different device will disrupt Sentry's ability to use it in order to download dSYM files.
-
-</Alert>
-
-These credentials will need to be verified by a two-factor authentication token that you can choose to have sent by SMS.
-
-**Re-authentication**
+**Invalid Credentials**
 
-The iTunes credentials and two-factor authentication are short-lived and need to be refreshed on a regular basis. Sentry will prompt users to re-authenticate in the **Issue Details** pages of a project, in the settings menu of a project, as well as in the "Custom Repositories" section of a project's "Debug Files" settings once their iTunes credentials expire.
+If your App Store Connect API key is revoked on the Apple side, the Sentry custom repository using this key's credentials will no longer be able to discover and fetch dSYM files and the following warnings will be displayed in the Senty's User Interface:
```

Comment:
I wonder how important it still is to show the screenshots given that this isn't a normal or often occurrence, I think just mentioning that it will stop working and you will get warnings in the UI might be enough.  But it doesn't do any harm though.

Independently I wonder if we should say something like "To update the credentials with new valid credentials click the 'configure' button"?

Reply:
I also think that it does not do any harm...screenshots are always good to have :)

About the "config button", I think an extra sentence informing the user about it will be good.. I'll add it